### ISSUE TITLE
How to reach 64% with FCN8sAtOnce?

### ISSUE THREAD

**Issue by sebgao** [wkentaro/pytorch-fcn#84]
I'm a little curious about how to reach 64% with FCN8sAtOnce model. In my experiments with downloaded same code (except for pytorch pretrained model instead of caffe), the best mean_iu within 100000 iterations is 59.04%. Any learning rate decay used here and other tricks? Thanks.

**Comment by wkentaro** [comment]
> except for pytorch pretrained model instead of caffe

I think this is why. The pytorch pretrained model is different from the one of caffe.

**Comment by Jinming-Su** [comment]
I conduct FCN8s_atonce with the same setting as this respository, while the miou at 96000 is 63.5
%, why?

**Comment by wkentaro** [comment]
I have no idea. Maybe pytorch default parameters can be changed in the latest version.
